### PR TITLE
Message model test

### PR DESCRIPTION
**Pull request by Ichi0302**
## what
I made text code of the message model
## why
To ensure that the message model works well

### CODE REVIEW

**Comment by Ichi0302** [comment]
https://gyazo.com/2c66c6a110dbf34875523cd6b9a38783

File: spec/factories/messages.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+
+  factory :message do
```
Comment (hzak2xx) [inline spec/factories/messages.rb:3]: faker使ってみよう

Comment (Ichi0302) [reply spec/factories/messages.rb:3]: done

**Pull request merged**